lib/ui/home/home_page.dart

```diff
@@ -0,0 +1,246 @@
+            zoomControlsEnabled: false,
+            onMapCreated: _onMapCreated,
+            initialCameraPosition:
+                CameraPosition(target: _center, zoom: 18.0, tilt: 70),
+          ),
+        ),
+        fastActionMenu(colorB1: colorB1, colorB3: colorB3, colorY1: colorY1),
+        ubicationCard(colorB1: colorB1, colorB2: colorB2),
+      ],
+    );
+  }
+}
+
+class ubicationCard extends StatelessWidget {
+  const ubicationCard({
+    super.key,
+    required this.colorB1,
+    required this.colorB2,
+  });
+
+  final Color colorB1;
+  final Color colorB2;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      alignment: Alignment.bottomCenter,
+      child: Padding(
+        padding: const EdgeInsets.fromLTRB(0, 0, 0, 50),
+        child: SizedBox(
+          width: 350.0,
+          height: 100.0,
+          child: Container(
+            alignment: Alignment.center,
+            decoration: BoxDecoration(
+              color: Colors.white,
+              borderRadius: BorderRadius.circular(5),
+            ),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: [
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    textFastActions(
+                        texto: "Cra. 1 #22-37, Bogotá",
+                        colorB1: colorB1,
+                        tamanioFuente: 15),
+                    textFastActions(
+                        texto: "25 puestos disponibles",
+                        colorB1: colorB2,
+                        tamanioFuente: 12),
+                  ],
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(30, 0, 5, 0),
+                  child: Container(
+                    alignment: Alignment.centerRight,
+                    child: Image.asset('assets/cuadrito.png', scale: 1.2),
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class fastActionMenu extends StatelessWidget {
+  const fastActionMenu({
+    super.key,
+    required this.colorB1,
+    required this.colorB3,
+    required this.colorY1,
+  });
+
+  final Color colorB1;
+  final Color colorB3;
+  final Color colorY1;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      alignment: Alignment.topCenter,
+      child: Padding(
+        padding: const EdgeInsets.fromLTRB(0, 80, 0, 0),
+        child: SizedBox(
+          width: 350.0,
+          height: 200.0,
+          child: Column(
+            children: [
+              Text(
+                'Buscar en:',
+                style: TextStyle(
+                  fontSize: 15,
+                  color: colorB1,
+                  decoration: TextDecoration.none,
+                ),
```

Comment:
Consider adding this TextStyle to [theme/theme_constants](https://github.com/ISIS3510-202320-Team13/Flutter/blob/main/lib/ui/theme/theme_constants.dart)

Reply:
Won´t use this style anymore